src/main/kotlin/xyz/atrius/waystones/data/config/Config.kt

```diff
@@ -15,8 +16,8 @@ class Config(plugin: KotlinPlugin) {
     }
 
     // The localization to be used
-    val localization: Property<Locale> =
-        Property("localization", Locale.ENGLISH, LocaleParser)
+    val locale: Property<Locale> =
+        Property("localization", Locale.ENGLISH, LocaleParser, { localization.reload() })
```

Comment:
I noticed you renamed this to `locale`, so you should also update `"localization"` to match the rename as that is the value that determines the property it saves and looks for in the config. Additionally it's preferable to keep the `onUpdate` outside the parenthesis as that is generally how kotlin handles function arguments.

Reply:
On second look I didn't see the 2nd commit which fixed this.